### DISCUSSION TITLE
Bug:Firefox:Chrome Memory consume grow infinitely in Gmail when dark mode

### DISCUSSION THREAD

**Discussion by yangu-hury** [darkreader/darkreader#8977] [General]
Darkreader makes the tab with Gmail open grow memory consume infinitely when user change from "inbox" to "trash" in dark mode. It can happen with other sites not mentioned only opening it tab or just doing some actions. Darkreader code logic of the injected code in sites should be revised to avoid this possibility.

**Comment by Gusted** [comment]
Please don't use discussions to report issues. Use [the issues](https://github.com/darkreader/darkreader/issues) section for that.
